src/modules/game.js

```diff
@@ -1,8 +1,41 @@
-class Game {
-  constructor(name, score) {
-    this.name = name;
-    this.score = score;
+const apiKey = 'gQxRXsSqZhbfITVQg4H7';
+const apiUrl = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${apiKey}/scores`;
+
+const postScore = () => {
+  const name = document.querySelector('#name').value;
+  const score = document.querySelector('#score').value;
+
+  fetch(apiUrl, {
+    method: 'Post',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      console.log('Success', data);
+    })
+    .catch((error) => {
+      console.error('Error', error);
+    });
+};
```

Comment:
Kindly make sure that you are using async and await Javascript functions. This is a project requirement. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/202337354-35b04951-5a1f-48e3-b204-c4a757418c33.png)